v22.2/cdc-transformations.md

```diff
@@ -0,0 +1,311 @@
+---
+title: Change Data Capture Transformations
+summary: Filter the changefeed data that emits to the changefeed sink.
+toc: true
+docs_area: stream
+---
+
+{% include feature-phases/preview.md %}
+
+{% include_cached new-in.html version="v22.2" %} Change data capture transformations allow you to define the change data emitted to your sink when you create a changefeed. The expression [syntax](#syntax) provides a way to select columns and apply filters to further restrict or transform the data in your [changefeed messages](changefeed-messages.html).  
+
+You can use CDC transformations to do the following:
+
+- Filter out specific rows and columns from changefeed messages to decrease the load on your downstream sink and support outbox workloads.
+- Modify data before it emits to reduce the time and operational burden of filtering or transforming data downstream.
+- Stabilize or customize the schema of your changefeed messages for increased compatibility with external systems.
+
+You can use any CockroachDB-supported SQL expression syntax that is not listed in [limitations](#limitations) to build a changefeed query.
+
+See the [Examples](#examples) section for further use cases.
+
+## Syntax
+
+There are two possible components to CDC transformations:
+
+- _Projections_ will select the columns that you want to emit data from. 
+- _Predicates_ will restrict the selected column change data based on the filters you apply.
```

Comment:
nit: these descriptions overlap just a little. Maybe if framed as input v output (i.e. modify the incoming data, then modify the outgoing data), something like:

> - _Projections_ limit the queried data to specific columns that you want to emit data from.
> - _Predicates_ limit the resulting column change data payload based on the filters you apply.